woocommerce/woocommerce-ios
# Store onboarding: WCPay setup celebration view

Closes: #10587 


## Description
- Presents a celebration view if user successfully sets up WCPay. 
- Adds dark mode image for WCPay badge image. 

## Testing instructions
- Create a new JN site with Jetpack and Woo. Set up your site with a WCPay-supported country (e.g. US).
- Follow the setup steps in P91TBi-4BH-p2 up to the step of installing the dev plugin only.
- Log in to the app and switch to the JN site.
- On the My Store screen, select Get Paid on the onboarding task list. 
- Select Begin Setup and finish Setup.
- Finish the setup in test mode. When the setup completes you should see the celebration view as a bottom sheet. 

## Screenshots

https://github.com/woocommerce/woocommerce-ios/assets/524475/17dac2cd-faa7-4df4-92b9-6b6e872985cb

<img width="294" alt="Screenshot 2023-08-31 at 1 19 43 PM" src="https://github.com/woocommerce/woocommerce-ios/assets/524475/437cf42c-cbe7-48fd-a52e-500288ae63d4">


Added dark mode Woo Pay badge image
| Dark | Light |
|--------|--------|
| ![Simulator Screen Shot - iPhone 14 Pro - 2023-08-31 at 13 14 38](https://github.com/woocommerce/woocommerce-ios/assets/524475/34b152e7-7df4-4cd2-824f-378be2f8391f) | ![Simulator Screen Shot - iPhone 14 Pro - 2023-08-31 at 13 14 43](https://github.com/woocommerce/woocommerce-ios/assets/524475/7002464d-cd01-4a4c-94b9-4016cd4402d7) | 

---
